src/components/Users/UserBanner.tsx

```diff
@@ -12,29 +14,36 @@ export default function UserBanner({ userData }: { userData: UserBase }) {
 
   return (
     <div className="mt-2 flex flex-col justify-between gap-3 sm:flex-row">
-      <div className="mb-3 flex w-full flex-col justify-between gap-3 rounded bg-white p-3 shadow-sm transition-all duration-200 ease-in-out sm:flex-row md:p-6">
+      <div className="mb-3 flex flex-col w-full justify-between gap-3 rounded p-3 shadow-sm transition-all duration-200 ease-in-out sm:flex-row md:p-6">
         <div className="flex flex-row gap-2 self-center">
           <Avatar
             imageUrl={userData?.profile_picture_url}
             name={formatDisplayName(userData)}
-            className="h-20 w-20 md:mr-2 lg:mr-3 lg:h-16 lg:w-16"
+            className="h-20 w-20 md:mr-2 lg:mr-3 lg:h-16 lg:w-16 shrink-0"
```

Comment:
`lg:mr-3 lg:h-16 lg:w-16`

Why? 🤔 

Let's keep the margin and size same here.